test/integration/Delegation.js

```diff
@@ -66,28 +66,20 @@ contract("Delegation", accounts => {
     it("registers transcoder 1 that self bonds", async () => {
         await token.approve(bondingManager.address, 1000, {from: transcoder1})
         await bondingManager.bond(1000, transcoder1, {from: transcoder1})
-
-        // Stake doesn't count until user joins transcoder pool
-        assert.equal(await bondingManager.getTotalBonded(), 0, "wrong total bonded")
-
         await bondingManager.transcoder(0, 5, {from: transcoder1})
-
-        assert.equal(await bondingManager.transcoderStatus(transcoder1), 1, "wrong transcoder status")
-        // Stake counted (+1000) after user joins transcoder pool
+        await roundsManager.mineBlocks(roundLength.toNumber() * 1)
```

Comment:
If I'm not mistaken this would cause the statement below to fail when calling initializeRound:

```
        // Check if already called for the current round
        require(lastInitializedRound < currRound);
```

Reply:
But we don't need to call `initializeRound()` here since transcoder registration is immediate and not impacted by round progression.